cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go

```diff
@@ -620,6 +618,20 @@ func TestValidateKubeconfigsForExternalCA(t *testing.T) {
 	// create a config with another server URL
 	configWithAnotherServerURL := setupdKubeConfigWithClientAuth(t, caCert, caKey, "https://4.3.2.1:4321", "test-cluster", "myOrg1")
 
+	// create a config with a ca cert containing multiple certificates
+	intermediateCaCert1, intermediateCaKey1 := certstestutil.SetupIntermediateCertificateAuthority(t, caCert, caKey)
+	intermediateCaCert2, intermediateCaKey2 := certstestutil.SetupIntermediateCertificateAuthority(t, intermediateCaCert1, intermediateCaKey1)
```

Comment:
```suggestion
	intermediateCACert1, intermediateCAKey1 := certstestutil.SetupIntermediateCertificateAuthority(t, caCert, caKey)
	intermediateCACert2, intermediateCAKey2 := certstestutil.SetupIntermediateCertificateAuthority(t, intermediateCACert1, intermediateCAKey1)
```

to use the proper camelCase for abbreviations